wiki/Skinning_Tutorial/osu!mania/en.md

```diff
@@ -1,106 +1,575 @@
-# Skinning Tutorial (osu!mania)
+# osu!mania
 
-## skin.ini
+This article assumes you are using [skin version](/wiki/skin.ini#versions) `2.5` or higher. Since the skin configuration heavily affects osu!mania skinning, this article will discuss both cases: using the default values (henceforth referred to as "basic") and modifying the skin configuration (henceforth referred to as "advanced").
```

Comment:
```suggestion
This tutorial uses [skin version](/wiki/skin.ini#versions) `2.5` or higher. Since the skin configuration heavily affects osu!mania skinning, this tutorial will discuss both cases: using the default values (henceforth referred to as "basic") and modifying the skin configuration (henceforth referred to as "advanced").
```